Replace `stripWeekday` in `StringHandlerService` with Date pipe

michael-small opened the pull request:
I saw the helper method `stripWeekday` from `StringHandlerService` and knew that it could be replaced by a date pipe. While more verbose, it is sure more readable than regex to the average person right out of UMM, like ourselves.

I remember making a big deal out of formatting dates in iteration 2 of 3601, and then Paul was like "BOOM! Pipes."

Docs: https://angular.io/api/common/DatePipe

Commit:
Replace `stripWeekday` in `StringHandlerService` with Date pipe

I saw the helper method `stripWeekday` from `StringHandlerService`
and knew that it could be replaced by a date pipe. While more
verbose, it is sure more readable than regex to the average person
right out of UMM, like ourselves.

I remember making a big deal out of formatting dates in iteration
2 of 3601, and then Paul was like "BOOM! Pipes."

Docs: https://angular.io/api/common/DatePipe

nkfoss commented:
Honestly, the StringHandler Service is such a joke. I created that way in the earlier stages, thinking it would be more useful. I've been thinking of scrapping it... there's just not enough justification for having the service (so far).

Maybe the next step for this is to move it into the component...? What do you think?

Commit:
Delete `StringHandlerService`, replace weekday regex w/date-pipe

With `stripWeekday` from the service removed in favor of pipes,
the service has been scrapped. There was also a `stripWeekday`
that was defined in `navbar-component` that has been removed
too.

CLose #47 (Scrap the string-handler service and convert other
string handling methods to piping)

Commit:
Merge branch 'master' of https://github.com/nkfoss/ohm-str into replaceStripWeekdayWithPipe

michael-small commented:
@nkfoss ready for review

nkfoss committed:
Merge branch 'master' into replaceStripWeekdayWithPipe
